packages/test/src/test-webpack.ts

```diff
@@ -0,0 +1,27 @@
+import { Worker } from '@temporalio/worker';
+import test from 'ava';
+import { v4 as uuid4 } from 'uuid';
+import { RUN_INTEGRATION_TESTS } from './helpers';
+
+if (RUN_INTEGRATION_TESTS) {
+  test('WorkerOptions.bundlerOptions.configureWebpack works', async (t) => {
+    const taskQueue = `${t.title}-${uuid4()}`;
+    await t.throwsAsync(
+      Worker.create({
+        taskQueue,
+        workflowsPath: require.resolve('./workflows'),
+        bundlerOptions: {
+          configureWebpack: (config) => {
```

Comment:
I'd use the declarative form to match the rest of the options (`webpackConfigHook`?)